apps/dotcms-ui/src/app/view/components/_common/forms/dot-push-publish-form/dot-push-publish-form.component.ts

```diff
@@ -170,7 +171,7 @@ export class DotPushPublishFormComponent
             });
     }
 
-    private loadFilters(): Observable<any> {
+    private loadFilters(): Observable<unknown> {
         return this.dotPushPublishFiltersService.get().pipe(
```

Comment:
I think we need to talk about when someone add type to a service needs to go where is used and add the type.